scripts/gha/integration_testing/xcodebuild.py

```diff
@@ -142,3 +83,19 @@ def _get_ios_env_from_target(ios_sdk):
     return "iphonesimulator"
   else:
     raise ValueError("Unrecognized ios_sdk: %s" % ios_sdk)
+
+
+def generate_unsigned_ipa(build_dir):
```

Comment:
There are a few undocumented assumptions/behaviours here, which should either be explained in the docstring or exposed as arguments.

- There's a subdirectory called "Debug-iphoneos" in the build dir, which implicitly assumes we ran xcodebuild with "Debug" configuration.
- "build_dir" is the directory containing Debug-iphoneos, and not Debug-iphoneos itself.
- The resulting ipa will be called "integration_test.ipa", and will be placed within Debug-iphoneos, not build_dir.

Reply:
Will add document, and change the variables' name.